tests/integration_tests/test_api.py

```diff
@@ -25,7 +25,6 @@
 
 
 # The following imports are pytest fixtures, required for running the tests
-from tests.fixtures.filenames import csv_filename
```

Comment:
please readd this, pycharm code optimization removed it because it's not referenced, but is actually needed

Reply:
Done.